src/lib/Layout/FormLayout/FormLayout.tsx

```diff
@@ -13,7 +13,7 @@ type Props = {
   id?: string
   label?: string
   labelOptional?: string
-  layout: 'horizontal' | 'vertical'
+  layout?: 'horizontal' | 'vertical'
```

Comment:
I think should `layout` should have a default value if it's being made optional so that we keep the expected behaviour that it is always one or the other

Reply:
Okay, makes sense.

Reply:
I will just keep it as required for now since it was required before, there's not really any need to change it, maybe we can have a default vertical later

Reply:
Actually, after changing this I just saw the errors in `Input.tsx`. In input.tsx, the components have the `layout` prop which is optional (Input and TextArea both), and both these components use the FormLayout component. Now, we either need to give a default layout in these 2 components or change the `layout` in FormLayout to be optional and give a default there.

Reply:
Oh yea, this might be confusing. 

We have a layout prop on data input components (Input, Select, Radio etc) so a user of the library can pass a preferred layout option `"horizontal"` or `"vertical"`.

the components always have a default (i think..) hence this wasn't optional in the FormLayout, to make sure that they always had layout prop set. I think it's usually horizontal - I guess if they're all horizontal then really FormLayout should have a default of horizontal, and the rest of the components don't need to set a default value.

Reply:
I'd probably opt for the FormLayout having a default, just to make it simpler. 

Although I think Toggle is the odd one out, because it uses `align` and it has a "vertical" 'layout`. So just check the components are looking ok with whatever changes you put in ;-)

Reply:
Alrighty, `layout` optional in FormLayout with a default `'horizontal'` works with all components I think (checked in storybook), including Toggle